hieradata_aws/class/integration/whitehall_db_admin.yaml

```diff
@@ -1,25 +1,25 @@
 govuk_env_sync::tasks:
-  "pull_whitehall_production_daily":
+  "pull_whitehall_staging_daily":
```

Comment:
You'll need to make "pull_whitehall_production_daily" ensure absent as part of this.